python/ray/experimental/state/common.py

```diff
@@ -12,7 +12,11 @@
 DEFAULT_RPC_TIMEOUT = 30
 DEFAULT_LIMIT = 100
 DEFAULT_LOG_LIMIT = 1000
-MAX_LIMIT = 10000
+# Max number of entries from API server to the client
+MAX_LIMIT_FROM_API_SERVER = 10 * 1000  # 10k
+# Max number of entries from data sources (rest will be truncated)
+# FIXME(rickyyx): should we make this configurable?
```

Comment:
yes!

Reply:
ideally, we should make it configurable from API call (MAX_LIMIT_FROM_API_SERVER=100 ray list actors) but it might be a big change. Can you see how trivial it is?